lib/cookie.js

```diff
@@ -5,7 +5,7 @@
  */
 
 var bindAll = require('bind-all');
-var clone = require('@ndhoule/clone');
+var clone = require('./utils/clone');
```

Comment:
Checking for regressions: 
- Cookies are created with the right options https://github.com/segmentio/analytics.js-core/blob/master/lib/cookie.js#L76

![image](https://user-images.githubusercontent.com/484013/80995761-a13e3780-8df3-11ea-928c-9309338c65d7.png)